projects/ad4630_fmc/common/adaq42xx_fmc.txt

```diff
@@ -0,0 +1,24 @@
+FMC_pin FMC_port    Schematic_name  System_top_name       IOSTANDARD Termination
+
+# adaq42xx
```

Comment:
Same comments like I said for projects/ad4630_fmc/common/ad463x_fmc.txt

Reply:
Done!